subsys/bluetooth/host/direction.c

```diff
@@ -155,6 +156,40 @@ static int hci_df_read_ant_info(uint8_t *switch_sample_rates,
 	return 0;
 }
 
+/* @brief Function handles send of HCI commnad to enable or disables CTE
+ *        transmission for given advertising set.
+ *
+ * @param[in] adv               Pointer to advertising set
+ * @param[in] enable            Enable or disable CTE TX
+ *
+ * @return Zero in case of success, other value in case of failure.
+ */
+static int hci_df_set_adv_cte_tx_enable(struct bt_le_ext_adv *adv,
+					bool enable)
+{
+	struct bt_hci_cp_le_set_per_adv_recv_enable *cp;
```

Comment:
bt_hci_cp_le_set_cl_cte_tx_enable

Reply:
Oh... sorry, how did it happen... Corrected to `bt_hci_cp_le_set_cl_cte_tx_enable`.